src/components/application_manager/src/commands/mobile/send_location_request.cc

```diff
@@ -60,29 +60,25 @@ void SendLocationRequest::Run() {
     return;
   }
 
-  smart_objects::SmartObject& msg_params = (*message_)[strings::msg_params];
-  if (msg_params.keyExists(strings::delivery_mode)) {
```

Comment:
@AKalinich-Luxoft 
Removing this part you break the requirement https://adc.luxoft.com/jira/browse/APPLINK-21910

Reply:
@dev-gh not yet. If this param will be disallowed by policy, then it will be cut off by CommandRequestImpl::RemoveDisallowedParameters() call before exact command execution, so this check seems redundant on this step